sdk/keyvault/azure-security-keyvault-jca/pom.xml

```diff
@@ -135,13 +136,20 @@
         <version>3.6.0</version> <!-- {x-version-update;org.apache.maven.plugins:maven-shade-plugin;external_dependency} -->
         <executions>
           <execution>
+            <id>minimize-shade</id>
             <goals>
               <goal>shade</goal>
             </goals>
             <configuration>
               <shadeTestJar>false</shadeTestJar>
               <createSourcesJar>${createSourcesJar}</createSourcesJar>
               <shadeSourcesContent>true</shadeSourcesContent>
+              <minimizeJar>true</minimizeJar>
```

Comment:
Why other dependencies can not apply `minimizeJar`?

Reply:
I think we do could do the same for others, but to minimize the changes in this PR, we probably could do it later.